Modules/QuestieAnnounce.lua

```diff
@@ -32,20 +28,24 @@ function QuestieAnnounce:AnnounceParty(questId, progressType, itemId, objectiveT
             else
                 objective = objectiveProgress.." "..objectiveText
             end
-            message = raidMarker .. " Questie : " .. l10n("%s for %s!", objective, questLink)
+            message = _GetAnnounceMarker() .. " Questie : " .. l10n("%s for %s!", objective, questLink)
         elseif progressType == "item" then
             local itemLink = select(2, GetItemInfo(itemId))
-            message = raidMarker .. " Questie : " .. l10n("Picked up %s which starts %s!", itemLink, questLink)
-        end
-
-        if (not message) or alreadySentBandaid[message] then
-            return
+            message = _GetAnnounceMarker() .. " Questie : " .. l10n("Picked up %s which starts %s!", itemLink, questLink)
         end
 
-        alreadySentBandaid[message] = true
+        _QuestieAnnounce:AnnounceParty(message, false)
+    end
+end
 
-        SendChatMessage(message, "PARTY")
+function _QuestieAnnounce:AnnounceParty(message, allowRepeat)
+    if (not message) or ((allowRepeat == nil or not allowRepeat) and alreadySentBandaid[message]) then
```

Comment:
```suggestion
    if (not message) or (not allowRepeat) and alreadySentBandaid[message]) then
```

The `not` operator returns `true` for everything that is falsy. So the condition before checked the same twice.

Reply:
True indeed, in my first draft it the flag was initially negated, so the condition made more sense.